client/components/RatingsAndReviews/Reviews/ReviewsList.jsx

```diff
@@ -5,7 +5,7 @@ import { Skeleton } from '@material-ui/lab';
 const ReviewsList = () => {
   return (
     <>
-    <p>248 reciews, sorted by relevance</p>
+    <p>248 reviews, sorted by relevance</p>
```

Comment:
Try experimenting with the ```<Typography>``` component material-ui provides, you can do stuff like ```<Typography variant="subtitle1"> 248 reviews, sorted by relevance<Typography />``` and it will automatically update to whatever themes we eventually decide on!

Reply:
Thanks for pointing this out! I knew there was a better way to handle text😅